apps/examples/testcase/le_tc/network/tc_net_bind.c

```diff
@@ -45,7 +45,10 @@ static void tc_net_bind_p(void)
 {
 	struct sockaddr_in sa;
 	int SocketFD = socket(PF_INET, SOCK_STREAM, IPPROTO_TCP);
-
+	if (SocketFD < 0) {
+		printf("Socket creation fail(%d)\n", __LINE__);
+		return;
```

Comment:
Let's use TC_ASSERT macro. Don't quit without any result reporting.

Reply:
If it was failing then we can't proceed any TC. so just printing fail log is enough.

Reply:
for example, In this testcase, what I want to test is bind() not socket(), so I think the result of socket() doesn't need to affect the result of TC. And most cases it would not fail. Unfortunately if socket creation failed then it means the preparation for test is wrong. So I just added log to get information.
Eventually we can get enough informations by adding these logs.